torchxrayvision/utils.py

```diff
@@ -73,3 +82,31 @@ def load_image(fname: str):
     img = img[None, :, :]
 
     return img
+
+def read_xray_dcm(path:Union[PathType, BinaryIO, DicomFileLike], voi_lut:bool=False, fix_monochrome:bool=True)->ndarray:
```

Comment:
I think having the input just be PathType is better for simplicity.

Also, this doesn't do any checks to make sure the file is what we expect. Aren't there some restrictions that can be put on the format based on what you tested it for?

Reply:
I will make it `PathType` only. 

The type annotation is for hinting and docstring generation only. I think `mypy` can be used to enforce consistency across the entire codebase.

In the second part of the comment are you referring to the validity of the DCM file?

Reply:
> In the second part of the comment are you referring to the validity of the DCM file?

Ya, in the sense that we should white list the types of images that are supported. Like the valid PhotometricInterpretations that are expected to work as well as other fields like Color Space, PixelRepresentation, and others I am not aware of.

Reply:
I think the most logical thing should be to leave that on pydicom. We could focus on checking modalities for `CR` or `DX` as that is most XRays are going to be in practice but still allow people to continue to use the loader(eg. Mammography) and raise some kind of warning for that.